File: Tests/NSArray+Sync_Tests.m
@import XCTest;

#import "NSArray+Sync.h"
#import "BaseTestCase.h"

@interface NSArray_Sync_Tests : BaseTestCase

@end

@implementation NSArray_Sync_Tests

#pragma mark Bug 125 => https://github.com/hyperoslo/Sync/issues/125

/*- (void)testChangesPreprocessing {
    NSDictionary *formDictionary = [self objectsFromJSON:@"bug-125-light.json"];
    NSString *uri = formDictionary[@"uri"];

    DATAStack *dataStack = [self dataStackWithModelName:@"Bug125"];

    NSDictionary *preprocessed = [@[formDictionary] preprocessForEntityNamed:@"Form"
                                                              usingPredicate:[NSPredicate predicateWithFormat:@"uri == %@", uri]
                                                                      parent:nil
                                                                   dataStack:dataStack].firstObject;

    XCTAssertEqualObjects(preprocessed, formDictionary);
}*/

@end

File: Tests/NSArray+SyncTests.swift
import XCTest

class NSArray_SyncTests: XCTestCase {
  // Bug 125 => https://github.com/hyperoslo/Sync/issues/125

  /*func testPreprocessForEntityNamed() {
    let formDictionary = Helper.objectsFromJSON("bug-125-light.json") as! [String : NSObject]
    let uri = formDictionary["uri"] as! String
    let dataStack = Helper.dataStack("Bug125")

    let preprocessed = ([formDictionary] as NSArray).preprocessForEntityNamed("Form", predicate: Predicate(format: "uri = %@", uri), parent: nil, dataStack: dataStack).first!  as! [String : NSObject]
    XCTAssertEqual(preprocessed, formDictionary)
  }*/
}
